.github/workflows/post_aura_gauge_votes.yaml

```diff
@@ -0,0 +1,41 @@
+name: Post vlAURA snapshot votes to voter multisig and send to the vote relayer
+
+on:
+  workflow_dispatch:
+    inputs:
+      week-string:
+        description: 'Week string that votes are are being posted. should be YYYY-W##'
+        required: true
+
+jobs:
+  post_aura_guage_votes:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: vlAURA Voting
+      env:
+        ETHNODEURL: ${{ secrets.ETHNODEURL }}
+        PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+      run: |
+        pwd
+        RUN_DIR=tools/python
+        pip3 install -r $RUN_DIR/requirements.txt
+        python3 $RUN_DIR/aura_snapshot_voting/vote.py --week-string "${{ github.event.inputs.week-string }}"
+    - name: Create PR
+      id: cpr
+      uses: peter-evans/create-pull-request@v5
+      with:
+        commit-message: "task: vlaura vote report"
+        title: "vlaura vote report for ${{ github.event.inputs.week-string }}"
```

Comment:
You can't use event inputs here, only env vars.  Check out the add_rewards pr/action for an example.

Also if you use branch-suffix: timestamp you'll already get a unique branch name, so don't need both.

Reply:
also add  detail into commit message, why not :)